docs/yaml/objects/compiler.yaml

```diff
@@ -180,7 +180,9 @@ methods:
 
 - name: alignment
   returns: int
-  description: Returns the alignment of the specified type.
+  description: |
+    Returns the alignment of the specified type. For C-like languages,
+    the header `stddef.h` is included implicitly.
```

Comment:
It also includes `stdio.h` for native compilation, should be same text as others, right?

Reply:
otoh, maybe we should only guarantee `stddef.h` in doc, because `stdio.h` is more an implementation detail to get printf?

Reply:
> It also includes `stdio.h` for native compilation, should be same text as others, right?

It should be better to mention both headers, otherwise using implicitly included <stdio.h> would be another UB, isn't it?